lib/environment/class-environment.php

```diff
@@ -68,4 +68,14 @@ public static function is_batch_container( $hostname, $env = array() ) {
 
 		return false;
 	}
+
+	public static function is_dev_env_container() {
+		if ( isset( $_ENV['LANDO_APP_ROOT'] ) ) {
+			if ( preg_match( '/^[\w\-. \/\-_]+vip\/dev-environment\/[\w\-. \-_]+$/', $_ENV['LANDO_APP_ROOT'] ) ) {
```

Comment:
I don't really think the regex is needed here - if the `$_ENV['LANDO_APP_ROOT']` we must be in Lando, right? :)

Reply:
> if the `$_ENV['LANDO_APP_ROOT']` we must be in Lando, right? :)

This was my thought at first, but then I realized that I don't really know if mu-plugins is being used in Lando based projects which are not our dev-env. 

Perhaps a user is loading mu-plugins in their own iteration of a Lando env, and maybe this change would not be welcome in such a scenario. 

Trying to use a scalplel and not a hatchet, but maybe it's overkill? Im not in love with the computation efficiency of it. What do you think about that?